spec.html

```diff
@@ -6747,6 +6781,16 @@ <h1>
           1. Perform ? DefineField(_O_, _fieldRecord_).
       </emu-alg>
     </emu-clause>
+
+    <emu-clause id="sec-evaluatestaticblock" aoid="EvaluateStaticBlock">
+      <h1>EvaluateStaticBlock ( _receiver_, _blockRecord_ )</h1>
+      <p>The abstract operation EvaluateStaticBlock takes arguments _receiver_ (an Object) and _blockRecord_ (a ClassStaticBlockDefinition Record). It performs the following steps when called:</p>
```

Comment:
```suggestion
    <emu-clause id="sec-evaluatestaticblock" type="abstract operation">
      <h1>
        EvaluateStaticBlock (
          _receiver_: an Object,
          _blockRecord_: a ClassStaticBlockDefinition Record,
        )
      </h1>
      <dl class="header">
      </dl>
```

Reply:
Ah, I didn't realize this was a thing. I'm a bit confused by the `<dl class="header">` tag though. I assume it provides non-printing metadata for the clause? Its a bit confusing to have two metadata formats in the spec HTML:

- `<pre class="metadata">` &mdash; Used at the top for document metadata.
- `<dl class="header">` &mdash; Used for clauses.

Also, It doesn't seem like the documentation at https://tc39.es/ecmarkup/#metadata has been updated to reflect this format.

Reply:
> I'm a bit confused by the `<dl class="header">` tag though. I assume it provides non-printing metadata for the clause (or somehow attaches extra info to the clause)?

Metadata yes (well, depending on your definition of "metadata"), non-printing no. Currently, I think the content of any `<dd>` that appears in a `dl.header` will also appear in the resulting rendered preamble. But in future, it might also contain non-printing info. 

> Also, It doesn't seem like the documentation at https://tc39.es/ecmarkup/#metadata has been updated to reflect this format.

Yeah, I think the closest it has is [the PR that added support for the new format](https://github.com/tc39/ecmarkup/pull/327).